terraspace/app/stacks/peer-kubernetes-components/specs/troubleshoot/curl.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
```

Comment:
I didn't see a reference to any of these `/spec/**.yaml` files within the terraform. I could have easily missed it.  Just wanted to understand the use case and for the `/troubleshoot/` specs, are they conditional?

Reply:
fThey really are not used in Terraform, but useful for quickly creating pods manually that can help you troubleshooting.
I kept them there so it could help other team members, but if you don't see value I can remove it.

Reply:
We don't have to block this PR for it.  I would say, if this is just for reference and not really related to terraform or terraspace, we should just put it in a subfolder under `docs/*` for "Troubleshooting" or something

Reply:
Done! Created an "utils" folder at root and moved the troubleshooting away from unrelated terraform structure.